src/Asset/TagRenderer.php

```diff
@@ -91,7 +91,7 @@ public function renderWebpackScriptTags(string $entryName, string $packageName =
                 $this->convertArrayToAttributes($attributes)
             );
 
-            $this->renderedFiles['scripts'][] = $attributes['src'];
+            $this->renderedFiles['scripts'][] = $attributes;
```

Comment:
Unfortunately, this is a BC break, as it changes the structure of what's returned from `getRenderedScripts()` and `getRenderedStyles()`.

Probably we need to add and maintain a second property - e.g. `$this->renderedFilesWithAttributes()`. Then, in `getRenderedScripts()` (and also for styles), we could add a new `bool $includeAttributes = false` argument. If that's true, then we return the `renderedFilesWithAttributes`.

Reply:
Hi @weaverryan , Sorry for the delay. Thanks for your comment.
I fixed the BC break according your recommendations, if you want to check again.

Reply:
@weaverryan @jrushlow This PR has been open for almost 2 years now, could either one spare some time to review it further and merge it? Thanks in advance.